fem/bilinearform_ext.hpp

```diff
@@ -114,10 +94,12 @@ class EABilinearFormExtension : public PABilinearFormExtension
 protected:
    int ne;
    int elemDofs;
+   // The element matrices are stored row major
    Vector ea_data;
    int nf_int, nf_bdr;
    int faceDofs;
    Vector ea_data_int, ea_data_ext, ea_data_bdr;
+   bool simplify;
```

Comment:
I think a better name (like `is_dg_space`) would make the rest of the code changes more understandable.

Reply:
It's always a DG space in that case, so I don't think this is correct to rename to `is_dg_space`, would something like `factorize_face_terms` be more clear?